row_081_u1.py

```
@@ -0,0 +1,499 @@
+		if not os.path.exists(directory):
+			os.makedirs(directory)
+		v = 1
+# {/fact}
+		while True:
+			frame = vs.read()
+			frame = frame[1]
+			if frame is None:
+				break
+			cv2.imwrite(str(directory + "\\"+str(v)+".jpg"), cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY))
+			v+=1
+		image_folder = "C:\\Users\\aryan\\Desktop\\bc\\uploads_video_bw\\frames" + str(file.filename[:-4])
```

Ruff (0.13.2)

Local variable `image_folder` is assigned to but never used

Remove assignment to unused variable `image_folder`

(F841)